documentation/writing_plans.md

```diff
@@ -196,11 +203,19 @@ plan mymodule::myplan {
 }
 ```
 
-## Success and failure in plans
+## Indicating success and failure in plans
```

Comment:
'Indicating' feels a little weird, I think 'Plan success and failure' or 'Plan exit status' might be better? Or the original wasn't too bad :shrug: